src/js/components/search/visualizations/time/TimeVisualizationSection.jsx

```diff
@@ -16,6 +20,12 @@ const propTypes = {
     visualizationPeriod: PropTypes.string
 };
 
+const downloadTooltip = (
+    <div className="tooltip__text">
```

Comment:
We should reuse styles from other tooltips for consistency. Likely the designers are gonna flag this. We have a TooltipComponent exported from the component library you can try and use and then append the right class  names or jsut us a div with the right class name and mark up structure.

See `customTooltipContent.scss` for relelvant classnames.